### PR TITLE
Small change for histogram-page

### PR DESCRIPTION
**Pull request by Aehrraid**
only one stylish margin.

### CODE REVIEW

**Comment by howardchung** [comment]
@albertcui should review

**Comment by Aehrraid** [comment]
It's only one margin ;)

**Comment by howardchung** [comment]
yeah, but I like to have @albertcui review all PRs unless fixing a critical bug.

**Comment by Aehrraid** [comment]
Ok!

File: views/player/player_histograms.jade [hunk 1]
```diff
@@ -7,7 +7,7 @@ block playercontent
         - var times={"duration":1,"first_blood_time":1}
         script.
           var histogramData = {};
-        p: button.btn.btn-xs.btn-dark.toggle-button.bright-border.collapsed(onclick="showAll()") Show All Histograms
+          p: button.btn.btn-xs.btn-dark.toggle-button.bright-border.collapsed(onclick="showAll()" style="margin-right: 3px") Show All Histograms
```
Comment (albertcui) [inline views/player/player_histograms.jade]: The p is inside the scrip tag. Otherwise, looks good.

Comment (Aehrraid) [inline views/player/player_histograms.jade]: The p really has to be inside of <script>. This sounds weird?!

Comment (howardchung) [inline views/player/player_histograms.jade]: html inside javascript?

Comment (Aehrraid) [inline views/player/player_histograms.jade]: The p really has to be inside of script? This sounds weird?!

Comment (howardchung) [inline views/player/player_histograms.jade]: it shouldn't be inside.  I think Albert wants it fixed before merge.

**Comment by howardchung** [comment]
@Aehrraid going to fix it?

**Comment by howardchung** [comment]
I actually moved the "show all" options to its own line, so this isn't needed anymore?

**Pull request closed without merging**